js/opencast/src/Paella/config/config.json

```
@@ -11,6 +11,11 @@
 
   "defaultAudioStream": "presenter",
 
+  "ui": {
+            "hideUITimer": 1000,
+            "hideOnMouseLeave": true
```

Indentation also here!
```suggestion
    "hideOnMouseLeave": true
```